libraries/tock-tbf/src/types.rs

```diff
@@ -60,6 +60,12 @@ pub enum TbfParseError {
     /// fail and that will trigger a different error.
     InternalError,
 
+    /// Found a non-Footer in the footer region.
+    InvalidFooter(usize),
+
+    /// Found a non-Header in the header region.
+    InvalidHeader(usize),
```

Comment:
Do we need these?

Reply:
Guess not -- better to remove this extra code. Will do.

Reply:
Garners some good savings. :)